core/src/main/resources/lib/hudson/progressive-text.js

```diff
@@ -28,6 +28,12 @@ Behaviour.specify(
         parameters: { start: e.fetchedBytes },
         requestHeaders: headers,
         onComplete: function (rsp) {
+          if (rsp.status >= 500) {
```

Comment:
```suggestion
          if (rsp.status >= 500 || rsp.status == 0) {
```

To support configurations without reverse proxy.

Reply:
I was not even considering configurations without a reverse proxy as an option to support. Could try.

Reply:
To be clear, the use case I am interested in is a K8s-like setup where the reverse proxy does not even forward traffic to Jenkins at all until it is fully up and running. So, you go from 200 to 5xx for a couple minutes and then back to 200. Supporting a “naked” server is not in scope, though of course I do not wish to make the experience in that scenario _worse_.